src/representations/representation_manager_spherical_expansion.hh

```diff
@@ -614,7 +614,9 @@ namespace rascal {
       }
       if (hypers.find("compute_gradients") != hypers.end()) {
         this->compute_gradients = hypers.at("compute_gradients").get<bool>();
-      } // Default false (don't compute gradients)
+      } else { // Default false (don't compute gradients)
+        this->compute_gradients = false;
```

Comment:
It's already initialized to false by the default constructor (where it's declared) -- does this actually change anything?